docs/product/accounts/membership.mdx

```diff
@@ -49,6 +49,8 @@ Organization-level roles enable broad access to the entire organization, as desc
 | Can transfer projects to another organization                                   |             |            |           |             | ✓         |
 | Can remove an organization                                                      |             |            |           |             | ✓         |
 
+\*For Business and Enterprise plans, Org Members who are not a Team Admin for any team can create a new personal project. When creating this project, the member can add a new personal team that they will become a Team Admin for. [Team Admin permissions](#permissions-1) will then apply for this team.
```

Comment:
```suggestion
\*For Business and Enterprise plans, even Org Members who are not a Team Admin for any team can create a new personal project. When creating this project, the member will get assigned a new personal team that they will become a Team Admin for. [Team Admin permissions](#permissions-1) will then apply for this team.
```

Reply:
Adding this change only because they don't create the team, we automatically create the team for them when the try to create a new project.

Reply:
Thanks for clarifying!